qrenderdoc/Windows/TextureViewer.ui

```diff
@@ -138,6 +138,9 @@
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
+      <property name="toolTip">
+       <string>Zoom Range</string>
```

Comment:
The tooltip should expand and explain what this button does, it doesn't have to be so short. For example:

> Zoom the visible range controls to the current black and white points.